app/bundles/LeadBundle/Form/Type/ContactFrequencyType.php

```diff
@@ -40,167 +42,32 @@ public function __construct(CoreParametersHelper $coreParametersHelper)
-                        );
-                        $type = 'datetime';
-                    } else {
-                        $attributes = array_merge(
-                            $attr,
-                            [
-                                'class' => 'form-control',
-                            ]
-                        );
-                        $type = 'date';
-                    }
-                    if (!$options['public_view'] || $showContactPauseDates) {
-                        $builder->add(
-                            'contact_pause_start_date_'.$channel,
-                            $type,
-                            [
-                                'widget'     => 'single_text',
-                                'label'      => false, //'mautic.lead.frequency.contact.start.date',
-                                'label_attr' => ['class' => 'text-muted fw-n label3'],
-                                'attr'       => $attributes,
-                                'format'     => 'yyyy-MM-dd',
-                                'required'   => false,
-                            ]
-                        );
-                        $builder->add(
-                            'contact_pause_end_date_'.$channel,
-                            $type,
-                            [
-                                'widget'     => 'single_text',
-                                'label'      => 'mautic.lead.frequency.contact.end.date',
-                                'label_attr' => ['class' => 'frequency-label text-muted fw-n label4'],
-                                'attr'       => $attributes,
-                                'format'     => 'yyyy-MM-dd',
-                                'required'   => false,
-                            ]
-                        );
-                    }
-                }
-            }
         }
 
-        if (!$options['public_view']) {
-            $builder->add(
-                'lead_lists',
-                'leadlist_choices',
-                [
-                    'label'      => 'mautic.lead.form.list',
-                    'label_attr' => ['class' => 'control-label'],
-                    'multiple'   => true,
-                    'expanded'   => $options['public_view'],
-                    'required'   => false,
-                ]
-            );
-        } elseif ($showContactSegments) {
-            $builder->add(
-                'lead_lists',
-                'leadlist_choices',
-                [
-                    'global_only' => true,
-                    'label'       => 'mautic.lead.form.list',
-                    'label_attr'  => ['class' => 'control-label'],
-                    'multiple'    => true,
-                    'expanded'    => $options['public_view'],
-                    'required'    => false,
-                ]
-            );
-        }
+        $builder->add(
+            'lead_lists',
+            'leadlist_choices',
```

Comment:
`LeadCategoryType::class`

Reply:
Will update

Reply:
Should be `LeadListType::class`, updated so.